app/controllers/credits_controller.rb

```diff
@@ -99,14 +99,11 @@ def generate_cost
   end
 
   def cost_per_credit
-    if @number_to_purchase < 10
-      500
-    elsif @number_to_purchase < 100
-      400
-    elsif @number_to_purchase < 1000
-      300
-    else
-      250
+    case @number_to_purchase
```

Comment:
Correct me if I am wrong, but wouldn't the overlap in these ranges cause issues?
I was thinking more:
```
case @number_to_purchase
  in 0...10 then 500
  in 11...100 then 400
  in 101...1000 then 300
  else 250
```

Reply:
@msarit Ranges with 3 dots don't include the upper bound (half-open intervals in math, i.e. `[lower, upper)`)

```ruby
(1..3).to_a
#=> [1, 2, 3]
(1...3).to_a
#=> [1, 2]
```

I could change it to use inclusive ranges (closed intervals, i.e. `[lower, upper]`)

```ruby
  in 0..9 then 500
  in 10..99 then 400
  in 100...999 then 300
  else 250
```

but I usually prefer the more "symmetric" version.